301-create-ad-forest-with-subdomain/nested/CreateAndPrepnewVM.json

```diff
@@ -92,7 +101,15 @@
     "vmOSDisk": "[concat(parameters('vmName'),'-managed-OSDisk')]",
     "vmDataDiskSize": 10,
     "imagePublisher": "MicrosoftWindowsServer",
-    "imageOffer": "WindowsServer"
+    "imageOffer": "WindowsServer",
+
+    "scriptFolderName": "scripts/",
+    "scriptName": "Configure-DSC.ps1",
+    "scriptDownLoad": "[uri(parameters('_artifactsLocation'), concat(variables('scriptFoldername'), variables('scriptName'), parameters('_artifactsLocationSasToken')))]",
+    
+    "UrlIsBlob": "[or(contains(parameters('_artifactsLocation'), 'blob.core.windows.net'), contains(parameters('_artifactsLocation'), 'blob.core.usgovcloudapi.net'))]",

```

Comment:
let's just move this one script file to the root of the sample and remove this logic...

Reply:
It is now recognized as a bug: https://github.com/MicrosoftDocs/azure-docs/issues/43816